File: src/main/java/nl/dannyjelsma/soundboardbot/commands/music/StopCommand.java
package nl.dannyjelsma.soundboardbot.commands.music;

import com.jagrosh.jdautilities.command.Command;
import com.jagrosh.jdautilities.command.CommandEvent;
import nl.dannyjelsma.soundboardbot.audio.SoundManager;

public class StopCommand extends Command {

    public StopCommand() {
        name = "stop";
    }

    @Override
    protected void execute(CommandEvent event) {

        SoundManager manager = SoundManager.getInstance();

        manager.getScheduler().stop();
        event.getChannel().sendMessage("Stopped and cleared the queue.").queue();
    }
}

File: src/main/java/nl/dannyjelsma/soundboardbot/commands/music/StopCommand.kt
package nl.dannyjelsma.soundboardbot.commands.music

import com.jagrosh.jdautilities.command.Command
import com.jagrosh.jdautilities.command.CommandEvent
import nl.dannyjelsma.soundboardbot.audio.SoundManager

class StopCommand : Command() {
    init {
        name = "stop"
    }

    override fun execute(event: CommandEvent) {
        SoundManager.scheduler.stop()
        event.channel.sendMessage("Stopped and cleared the queue.").queue()
    }
}